libexec/azhpc-build.sh

```diff
@@ -303,6 +321,10 @@ for resource_name in $(jq -r ".resources | keys | @tsv" $config_file); do
             if [ "$resource_lowpri" = "true" ]; then
                 lowpri_option="--priority Low"
             fi
+            ppg_option=
+            if [ "$ppg" = "true" ]; then
+               ppg_option="--ppg $ppg_name"
```

Comment:
this will fail here if there is no proximity group name - should we check and give an error?

Reply:
Added an error check.